src/toil/server/wes/abstract_backend.py

```diff
@@ -255,7 +255,11 @@ def collect_attachments(self, run_id: Optional[str], temp_dir: Optional[str]) ->
             self.log_for_run(run_id, "Using workflow_url '%s'" % body.get("workflow_url"))
         else:
             raise MalformedRequestException("Missing 'workflow_url' in submission")
+
         if "workflow_params" not in body:
             raise MalformedRequestException("Missing 'workflow_params' in submission")
+        # TODO: make sure workflow_url and body["workflow_params"] does not access files outside of its own
+        #  workflow directory.
+        # this is hard for WDL files especially, unless we interpret every string to be a path
```

Comment:
I don't think this is too important; we aren't meant to provide multi-tenant security here.